components/scream/scripts/utils.py

```diff
@@ -398,3 +399,33 @@ def checkout_git_ref(git_ref, verbose=False, repo=None):
             print("  Switched to '{}' ({})".format(git_ref,git_commit))
             print_last_commit(git_ref=git_ref)
 
+###############################################################################
+def get_git_toplevel_dir(repo=None):
+###############################################################################
+    """
+    Get repo toplevel directory
+    """
+    stat, dirname = run_cmd_no_fail("git rev-parse --show-toplevel")
```

Comment:
run_cmd_no_fail does not return a stat, it only return the output and raises an expect if the command failed.

```
    return run_cmd_no_fail("git rev-parse --show-toplevel"
```

Reply:
True. Thanks.